frameworks/Java/greenlightning/src/main/java/com/ociweb/gl/benchmark/JSONBehaviorInstance.java

```diff
@@ -0,0 +1,39 @@
+package com.ociweb.gl.benchmark;
+
+import com.ociweb.gl.api.GreenRuntime;
+import com.ociweb.gl.api.HTTPRequestReader;
+import com.ociweb.gl.api.HTTPResponseService;
+import com.ociweb.gl.api.RestListener;
+import com.ociweb.json.encode.JSONRenderer;
+import com.ociweb.pronghorn.network.config.HTTPContentTypeDefaults;
+
+public class JSONBehaviorInstance implements RestListener {
+
+	byte[] messagePayload;
+
+	private static final JSONRenderer<JSONBehaviorInstance> renderJSON = new JSONRenderer<JSONBehaviorInstance>()
+			.startObject()
+				.string("message", (o,t) -> t.write(o.messagePayload) )
+			.endObject();
```

Comment:
https://frameworkbenchmarks.readthedocs.io/en/latest/Project-Information/Framework-Tests/#json-serialization Instantiation of the object that's going to be serialized has to happen on every request.

Reply:
I am adding a temp object and will resubmit